cloudflare/resource_cloudflare_standalone_healthcheck.go

```diff
@@ -0,0 +1,396 @@
+package cloudflare
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/pkg/errors"
+)
+
+func resourceCloudflareStandaloneHealthcheck() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareStandaloneHealthcheckCreate,
+		Read:   resourceCloudflareStandaloneHealthcheckRead,
+		Update: resourceCloudflareStandaloneHealthcheckUpdate,
+		Delete: resourceCloudflareStandaloneHealthcheckDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareStandaloneHealthcheckImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"zone_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
Probably doesn't matter if the name changes that much

```suggestion
```

Reply:
Indeed. I was under the false impression we can't update the name.